contrib/scalar/scalar.c

```diff
@@ -178,6 +196,27 @@ static int set_recommended_config(int reconfigure)
 		}
 	}
 
+	/*
+	 * The `log.excludeDecoration` setting is special because we want to
+	 * set multiple values.
+	 */
+	if (get_config_get_string("log.excludeDecoration", &value)) {
```

Comment:
```suggestion
	if (git_config_get_string("log.excludeDecoration", &value)) {
```

Suggested by the broken build.

Reply:
Right, I am on it, locally, but I see a test failure and am investigating before I will push.